src/main/scala/centaur/test/Test.scala

```diff
@@ -247,6 +276,29 @@ object Operations {
   def validateNoCacheHits(metadata: WorkflowMetadata, workflowName: String): Test[Unit] = validateCacheResultField(metadata, workflowName, "Cache Hit")
   def validateNoCacheMisses(metadata: WorkflowMetadata, workflowName: String): Test[Unit] = validateCacheResultField(metadata, workflowName, "Cache Miss")
 
+  def validateSubmitFailure(workflow: Workflow,
+                            expectedSubmitResponse: SubmitResponse,
+                            actualSubmitResponse: SubmitResponse): Test[Unit] = {
+    new Test[Unit] {
+      override def run: Try[Unit] = {
+        Try {
+          if (expectedSubmitResponse == actualSubmitResponse) {
+            ()
+          } else {
+            throw new RuntimeException(
+              s"""|Expected
```

Comment:
Since you're multi-lining anyway, might as well put the `Expected` on a new line so it's easier to see amongst all the stack tracings